firebase.json

```diff
@@ -69,8 +69,18 @@
         "type": 301
       },
       {
-        "source": "/get-started/",
-        "destination": "/setup/",
+        "source": "/getting-started",
+        "destination": "/get-started/",
+        "type": 301
+      },
+      {
+        "source": "/setup",
+        "destination": "/get-started/install/",
+        "type": 301
+      },
+      {
+        "source": "/setup-ide",
```

Comment:
This page doesn't exist. You probably meant https://flutter.io/ide-setup/.

Reply:
Good catch, fixed